contributors/design-proposals/bootstrap-discovery.md

```diff
@@ -138,12 +144,11 @@ The following keys are on the secret data:
 * **token-id**. As defined above.
 * **token-secret**. As defined above.
 * **expiration**. After this time the token should be automatically deleted.  This is encoded as an absolute UTC time using RFC3339.
-* **usage-bootstrap-signing**. Set to `true` to indicate this token should be used for signing bootstrap configs.  If omitted or some other string, it defaults to `false`.
+* **usage-bootstrap-signing**. Set to `true` to indicate this token should be used for signing bootstrap configs.  If this is missing from the token secret or set to any other value, the usage is not allowed.
+* **usage-bootstrap-authentication**. Set to true to indicate that this token should be used for authenticating to the API server.  If this is missing from the token secret or set to any other value, the usage is not allowed.  The bootstrap token authenticagtor will use this token to auth as a user that is `system:bootstrap:<token-id>` in the group `system:bootstrappers`.
 
```

Comment:
I'm in favor of generic free form description fields.  I'm happy to add it.  It is a bit of an overload -- this could go in an annotation or it could go in the of the secret.